public/javascript/APIHandler.js

```diff
@@ -3,23 +3,58 @@ class APIHandler {
     this.BASE_URL = baseUrl;
   }
 
-  getFullList () {
-
-  }
-
-  getOneRegister () {
-
-  }
-
-  createOneRegister () {
-
-  }
-
-  updateOneRegister () {
-
-  }
-
-  deleteOneRegister () {
-
-  }
+  getFullList() {
```

Comment:
Nice implementation of the solutions. However, for all your solutions, the responses from ```.then()``` aren't being used. At the moment, you are only logging them out on the console. You will need to make sure they are represented on the browser